docs/Config_Reference.md

```diff
@@ -3355,13 +3355,14 @@ run_current:
 
 ### [tmc2240]
 
-Configure a TMC2240 stepper motor driver via SPI bus. To use this
+Configure a TMC2240 stepper motor driver via SPI bus or UART. To use this
 feature, define a config section with a "tmc2240" prefix followed by
 the name of the corresponding stepper config section (for example,
 "[tmc2240 stepper_x]").
 
 ```
 [tmc2240 stepper_x]
+interface: spi
```

Comment:
This line added to Config_Reference.md seems to no longer be accurate.

-Kevin